<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReservasiWindow</class>
 <widget class="QMainWindow" name="ReservasiWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manajemen Reservasi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <property name="styleSheet">
       <string>QWidget { background-color: #2c3e50; }</string>
      </property>
      <layout class="QVBoxLayout" name="sidebarLayout">
       <item>
        <widget class="QLabel" name="logoLabel">
         <property name="styleSheet">
          <string>QLabel { color: white; font-size: 18px; font-weight: bold; padding: 15px; background-color: #2c3e50; }</string>
         </property>
         <property name="text">
          <string>üè®HOTEL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dashboardBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reservasiBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #34495e; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>Reservasi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="kamarBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>Kamar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pelangganBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>Pelanggan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pembayaranBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>Pembayaran</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="laporanBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #2c3e50; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #3498db; }</string>
         </property>
         <property name="text">
          <string>Laporan</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>150</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logoutBtn">
         <property name="styleSheet">
          <string>QPushButton { background-color: #e74c3c; color: white; border: none; text-align: left; padding-left: 20px; font-weight: bold; } QPushButton:hover { background-color: #c0392b; }</string>
         </property>
         <property name="text">
          <string>Logout</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="contentArea" native="true">
      <layout class="QVBoxLayout" name="contentLayout">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="styleSheet">
          <string>QLabel { font-size: 20px; font-weight: bold; color: #2c3e50; margin-left: 20px; padding-top: 10px; }</string>
         </property>
         <property name="text">
          <string>Data Reservasi</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QLineEdit" name="searchInput">
           <property name="placeholderText">
            <string>Cari Reservasi...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="filterStatus">
           <item>
            <property name="text">
             <string>Semua Status</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dipesan</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dibayar</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Batal</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refreshBtn">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="tambahBtn">
           <property name="text">
            <string>Tambah</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="reservasiTable">
         <property name="rowCount">
          <number>5</number>
         </property>
         <property name="columnCount">
          <number>8</number>
         </property>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nama</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Kamar</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Masuk</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Keluar</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Harga</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Aksi</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>Ahmad Fauzi</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>101</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>2025-10-20</string>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>2025-10-21</string>
          </property>
         </item>
         <item row="0" column="5">
          <property name="text">
           <string>300000</string>
          </property>
         </item>
         <item row="0" column="6">
          <property name="text">
           <string>Dipesan</string>
          </property>
         </item>
         <item row="0" column="7">
          <property name="text">
           <string>Edit</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>Dina Putri</string>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>102</string>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string>2025-10-21</string>
          </property>
         </item>
         <item row="1" column="4">
          <property name="text">
           <string>2025-10-23</string>
          </property>
         </item>
         <item row="1" column="5">
          <property name="text">
           <string>1000000</string>
          </property>
         </item>
         <item row="1" column="6">
          <property name="text">
           <string>Dipesan</string>
          </property>
         </item>
         <item row="1" column="7">
          <property name="text">
           <string>Edit</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>Rudi Hartono</string>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string>103</string>
          </property>
         </item>
         <item row="2" column="3">
          <property name="text">
           <string>2025-10-22</string>
          </property>
         </item>
         <item row="2" column="4">
          <property name="text">
           <string>2025-10-23</string>
          </property>
         </item>
         <item row="2" column="5">
          <property name="text">
           <string>750000</string>
          </property>
         </item>
         <item row="2" column="6">
          <property name="text">
           <string>Dibayar</string>
          </property>
         </item>
         <item row="2" column="7">
          <property name="text">
           <string>Edit</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>Lina Kurnia</string>
          </property>
         </item>
         <item row="3" column="2">
          <property name="text">
           <string>104</string>
          </property>
         </item>
         <item row="3" column="3">
          <property name="text">
           <string>2025-10-24</string>
          </property>
         </item>
         <item row="3" column="4">
          <property name="text">
           <string>2025-10-26</string>
          </property>
         </item>
         <item row="3" column="5">
          <property name="text">
           <string>1800000</string>
          </property>
         </item>
         <item row="3" column="6">
          <property name="text">
           <string>Dipesan</string>
          </property>
         </item>
         <item row="3" column="7">
          <property name="text">
           <string>Edit</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>Taufik Hidayat</string>
          </property>
         </item>
         <item row="4" column="2">
          <property name="text">
           <string>105</string>
          </property>
         </item>
         <item row="4" column="3">
          <property name="text">
           <string>2025-10-25</string>
          </property>
         </item>
         <item row="4" column="4">
          <property name="text">
           <string>2025-10-27</string>
          </property>
         </item>
         <item row="4" column="5">
          <property name="text">
           <string>3000000</string>
          </property>
         </item>
         <item row="4" column="6">
          <property name="text">
           <string>Dibayar</string>
          </property>
         </item>
         <item row="4" column="7">
          <property name="text">
           <string>Edit</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="footerLayout">
         <item>
          <widget class="QLabel" name="totalLabel">
           <property name="text">
            <string>Total: 5 Reservasi</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="pageLabel">
           <property name="text">
            <string>Halaman 1 dari 1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
